src/vmm/src/signal_handler.rs

```diff
@@ -94,7 +94,8 @@ fn get_metric_and_exitcode(signo: c_int) -> Option<(&'static dyn IncMetric, i32)
         SIGXCPU => Some((&METRICS.signals.sigxcpu, super::FC_EXIT_CODE_SIGXCPU as i32)),
         SIGBUS => Some((&METRICS.signals.sigbus, super::FC_EXIT_CODE_SIGBUS as i32)),
         SIGSEGV => Some((&METRICS.signals.sigsegv, super::FC_EXIT_CODE_SIGSEGV as i32)),
-        SIGPIPE => Some((&METRICS.signals.sigpipe, super::FC_EXIT_CODE_SIGPIPE as i32)),
+        // Dummy entry, never going to exit due to SIGPIPE.
+        SIGPIPE => Some((&METRICS.signals.sigpipe, super::FC_EXIT_CODE_OK as i32)),
```

Comment:
I think it would be cleaner to have an `Option` as the exit code

Reply:
Done, PTAL.